lib/templates/deploy.batch.dnx.consoleapp.template

```diff
@@ -0,0 +1,21 @@
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Deployment
+:: ----------
+
+echo Handling DNX Console Application deployment.
+
+:: 1. Set WebJob deployment target
+IF NOT DEFINED WEBJOB_DEPLOYMENT_TARGET (
+  SET WEBJOB_DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET%\App_Data\jobs\continuous\deployedJob
+)
+
+:: 2. Create WebJob directory
+IF EXIST "WEBJOB_DEPLOYMENT_TARGET" (
+  CALL :ExecuteCmd mkdir "%WEBJOB_DEPLOYMENT_TARGET%"
+)
+
+:: 3. KuduSync
+CALL :ExecuteCmd "%KUDU_SYNC_CMD%" -v 50 -f "{WEBJOB_PROJECT_PATH}" -t "%WEBJOB_DEPLOYMENT_TARGET%" -n "%NEXT_MANIFEST_PATH%" -p "%PREVIOUS_MANIFEST_PATH%" -i ".git;.hg;.deployment;deploy.cmd"
```

Comment:
One thing that's a bit funky is that the manifest ends up listing files relative to the WebJobs folder instead of wwwroot. e.g. in your sample, it has:
```
Program.cs
Project.json
.git
```

So if you switch from that content to (say) a static file, the manifest is broken.

Not sure if same issue exists with existing .net Console app support.